tests/dummy/app/pods/select/template.hbs

```diff
@@ -0,0 +1,241 @@
+<div class="section">
```

Comment:
Update on-change to onChange, there are a couple of places that need to be changed.